projects/dotcms-field-elements/src/components/dot-key-value/dot-key-value.e2e.ts

```diff
@@ -1,168 +1,423 @@
+
+            it('should not set disabled to child', async () => {
+                const form = await getForm();
+                const list = await getList();
+
+                expect(form.getAttribute('disabled')).toBeNull();
+                expect(list.getAttribute('disabled')).toBeNull();
+            });
+        });
+
+        describe('hint', () => {
+            it('should render', async () => {
+                element.setProperty('hint', 'Some hint');
+                await page.waitForChanges();
+
+                const hint = await dotTestUtil.getHint(page);
+                expect(hint.innerText).toBe('Some hint');
+                expect(hint.getAttribute('id')).toBe('hint-some-hint');
+            });
+
+            it('should not render', async () => {
+                const hint = await dotTestUtil.getHint(page);
+                expect(hint).toBeNull();
+            });
+
+            it('should handle invalid', async () => {
+                element.setProperty('hint', { a: 'object' });
+                await page.waitForChanges();
+
+                const hint = await dotTestUtil.getHint(page);
+                expect(hint).toBeNull();
+            });
+        });
+
+        describe('label', () => {
+            it('should render', async () => {
+                element.setProperty('label', 'Some label');
+                await page.waitForChanges();
+
+                const dotLabel = await dotTestUtil.getDotLabel(page);
+                expect(dotLabel.getAttribute('label')).toBe('Some label');
+            });
+
+            it('should not render', async () => {
+                const dotLabel = await dotTestUtil.getDotLabel(page);
+                expect(dotLabel.getAttribute('label')).toBe('');
+            });
+
+            it('should handle invalid', async () => {
+                element.setProperty('label', ['some', 'array']);
+                await page.waitForChanges();
+
+                const dotLabel = await dotTestUtil.getDotLabel(page);
+                expect(dotLabel.getAttribute('label')).toBe('some,array');
+            });
+        });
+
+        describe('name', () => {
+            it('should render', async () => {
+                element.setProperty('name', 'Some name');
+                await page.waitForChanges();
+
+                const dotLabel = await dotTestUtil.getDotLabel(page);
+                expect(dotLabel.getAttribute('name')).toBe('Some name');
+            });
+
+            it('should not render', async () => {
+                const dotLabel = await dotTestUtil.getDotLabel(page);
+                expect(await dotLabel.getAttribute('name')).toBe('');
+            });
+
+            it('should handle invalid', async () => {
+                element.setProperty('name', NaN);
+                await page.waitForChanges();
+
+                const dotLabel = await dotTestUtil.getDotLabel(page);
+                expect(dotLabel.getAttribute('name')).toBeNull();
+            });
```

Comment:
what happen with a not string value?

Reply:
I'm passing `NaN` that a not string value.